app/controllers/api/v2/applications_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V2::ApplicationsController < ActionController::API
+  def index
+    render json: { "applications": applications }
```

Comment:
I don't think these double quotes are needed?

`{ applications: applications }` should be OK, right?

Reply:
Yep, I spaced, fixing this is an upcoming PR.